cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Good point. Here are my thoughts about the matter:

I added this `.editorconfig` because I wanted to treat the cli as "kind of independent project with his own rules" but now that you pointed out I believe it is ok if I remove it and we start using the root one.

My only concern and this is one that I will not easily grant/giveaway is that I love the two spaces. LOL. Yes. I remember a discussion one or two years ago about the matter because the web is going in that front instead of the 4 spaces (web dev, not other like .NET where the 4 spaces rocks!). I remember that I forced myself to push the web standards and start using the 2 spaces. An eventually really loved it.

So, recap. My suggestion is to have only one `.editorconfig` at root and start using the 2 spaces for indentation. If I have greenlight from the team I will leave this `.editorconfig` as is and create a separate ticket to do those changes.

Thoughts, opinions, ideas? @juampick @mravinale @AlphaGit